habitat-baselines/habitat_baselines/config/experiments_hab3/socialnav_human_robot_oracle.yaml

```diff
@@ -0,0 +1,194 @@
+    - add_virtual_keys_base
+  - /habitat_baselines/rl/policy/obs_transforms@habitat_baselines.rl.policy.agent_1.obs_transforms.add_virtual_keys:
+    - add_virtual_keys_base
+  - /habitat_baselines/rl/policy@habitat_baselines.rl.policy.agent_0: hl_fixed # monolithic
+  - /habitat_baselines/rl/policy@habitat_baselines.rl.policy.agent_1: hl_fixed #hl_fixed #hab3_hl_neural
+  - /habitat_baselines/rl/policy/hierarchical_policy/defined_skills@habitat_baselines.rl.policy.agent_0.hierarchical_policy.defined_skills: oracle_skills_ma
+  - /habitat_baselines/rl/policy/hierarchical_policy/defined_skills@habitat_baselines.rl.policy.agent_1.hierarchical_policy.defined_skills: oracle_skills_ma_humanoid
+  - /habitat/task/measurements:
+    - social_nav_reward
+  - /habitat/task/lab_sensors:
+    - relative_resting_pos_sensor
+    - target_start_sensor
+    - goal_sensor
+    - joint_sensor
+    - is_holding_sensor
+    - end_effector_sensor
+    - target_start_gps_compass_sensor
+    - target_goal_gps_compass_sensor
+    - localization_sensor
+    - humanoid_joint_sensor
+  - /habitat/task/actions@habitat.task.actions.agent_0_arm_action: arm_action
+  - /habitat/task/actions@actions.agent_0_oracle_nav_action: oracle_nav_action
+  - /habitat/task/actions@actions.agent_1_oracle_nav_action: oracle_nav_action
+  - override /habitat/task/rearrange: multi_agent_tidy_house_base
+  # For Spot observation space
+  - override /habitat/simulator/agents@habitat.simulator.agents.agent_0: rgbd_head_rgbd_arm_agent
+  - _self_
+
+habitat:
+  task:
+    reward_measure: social_nav_reward
+    slack_reward: 0.
+    success_reward: 0.
+    task_spec: multi_agent_social_nav_solved
+    actions:
+      agent_0_arm_action:
+        grip_controller: MagicGraspAction
+      agent_0_oracle_nav_action:
+        agent_index : 0
+        spawn_max_dist_to_obj: -1.0
+      agent_1_oracle_nav_action:
+        motion_control: human_joints
+        spawn_max_dist_to_obj: 1.7
+    measurements:
+      cooperate_subgoal_reward:
+        end_on_collide: False
+
+  environment:
+    max_episode_steps: 700
+
+  gym:
+    obs_keys:
+      - agent_0_head_depth
+      - agent_0_relative_resting_position
+      - agent_0_obj_start_sensor
+      - agent_0_obj_goal_sensor
+      - agent_0_obj_start_gps_compass
+      - agent_0_obj_goal_gps_compass
+      - agent_0_joint
+      - agent_0_is_holding
+      - agent_0_ee_pos
+      - agent_0_localization_sensor
+      - agent_1_head_depth
+      - agent_1_relative_resting_position
+      - agent_1_obj_start_sensor
+      - agent_1_obj_goal_sensor
+      - agent_1_obj_start_gps_compass
+      - agent_1_obj_goal_gps_compass
+      - agent_1_is_holding
+      - agent_1_ee_pos
+      - agent_1_localization_sensor
+      - agent_1_humanoid_joint_sensor
+  simulator:
+    agents:
+      agent_0:
+        radius: 0.3
+        articulated_agent_urdf: data/robots/hab_spot_arm/urdf/hab_spot_arm.urdf
+        articulated_agent_type: "SpotRobot"
+        sim_sensors:
```

Comment:
Can you make a Hydra config for Spot so we can load these configurations easily into other tasks?

Reply:
thank you! Done!